hpx/lcos/server/queue.hpp

```diff
@@ -34,14 +35,14 @@ namespace hpx { namespace lcos { namespace server
     template <typename ValueType, typename RemoteType>
     class queue
       : public lcos::base_lco_with_value<ValueType, RemoteType>
-      , public components::component_base<queue<ValueType, RemoteType> >
+      , public components::managed_component_base<queue<ValueType, RemoteType> >
```

Comment:
All components which are LCOs (which are derived from base_lco) have to be managed components.